### ISSUE TITLE
Select Target OS for Driver Download

### ISSUE THREAD

**Issue by RareWare** [timsutton/brigadier#17]
With the release of Windows 10 Drivers the software always grabs the latest version. This latest version will not install with the installer on Windows 7 x64 with the tested model (Mac Mini Mid 2012). Can we specify a target OS version?

**Comment by timsutton** [comment]
I'm curious, are you getting an error about a missing DLL? This is what I get on 2013-era iMacs:

![2015-08-17 11 42 59](https://cloud.githubusercontent.com/assets/119358/9521304/1196d06c-4c9c-11e5-84b2-5a170b1cfc63.jpg)

**Comment by RareWare** [comment]
Yes, I get the same issue. Trying to figure out a solution for my workflow now that I have a moment. Will let you know what I end up doing.

**Comment by sunmetis** [comment]
Hi ... How to install last bootcamp version under windows with this app (brigadier)

**Comment by Bennow** [comment]
The same problem. Any updates or is there a solution?

**Comment by dire-alpaca** [comment]
Just doing a followup on this.

Running Brigadier on a Macmini7,1 with Win 7x64 and found it downloads Bootcamp 6 (6.0.6136) vs 5 (5.1.5769). Ye' ol windows 7 only supports 5.

However, I don't know if Apple has good data to see what versions of Bootcamp are made for what versions MS Windows. This may just be a limitation.

Should work great on Win8.1 and Win10 (which I should be deploying anyways ;)). I would argue since Apple is dropping support for older MS OSs, it shouldn't be a priority to resolve.

**Comment by nathanmcnulty** [comment]
I see a couple of ways of handling this, but I don't feel either one is worth spending the time on. First option is check for Windows 7, and if found, parse these pages until you end up with the correct download:

https://support.apple.com/en-us/HT205016

The second method would be to save the array of compatible ESD's, download the newest, extract, check, and if it's version 6 on Windows 7, delete and choose the next most recent (and keep doing this until you end up with one that is version 5 for 64bit installs or 4 for 32bit installs).
